atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTableTest.java

```diff
@@ -348,6 +357,80 @@ public void createDoesNotThrowWhenOverflowColumnAlreadyExistsAndAlterIsRan() thr
         verifyTableAltered();
     }
 
+    @Test
+    public void testIndexStrategy_conservative_sweep() {
```

Comment:
nit: in AtlasDB the general code conventions are to say what behaviour the test confirms - here `conservativeSweptTablesHaveCompressionStrategyTwo` or something like that